rust/connlib/tunnel/src/device_channel/tun_windows.rs

```diff
@@ -20,7 +20,8 @@ pub struct Tun {
 impl Tun {
     pub fn new(config: &InterfaceConfig) -> Result<Self> {
         const TUNNEL_UUID: &str = "e9245bc1-b8c1-44ca-ab1d-c6aad4f13b9c";
-        const TUNNEL_NAME: &str = "Firezone Tunnel";
+        // wintun automatically appends " Tunnel" to this
+        const TUNNEL_NAME: &str = "Firezone";
```

Comment:
this is the iface name? can we use the same name we use in `tun_linux`?

Reply:
Yeah this is what shows up in Device Manager. I think wintun.dll itself puts the " Tunnel" after it, I don't see it in the Rust wrapper: https://github.com/nulldotblack/wintun/blob/main/src/adapter.rs#L92

So if I put "tun-firezone" it'd show up as "tun-firezone Tunnel".

(It would be under here if I had it running)

![image](https://github.com/firezone/firezone/assets/13400041/5d2af3f1-bf7a-4262-bf76-cced8aa29616)